diff --git a/tensorflow/lite/tools/make/build_aarch64_lib.sh b/tensorflow/lite/tools/make/build_aarch64_lib.sh
index 0ce4089c11..8a3b2fef3e 100755
--- a/tensorflow/lite/tools/make/build_aarch64_lib.sh
+++ b/tensorflow/lite/tools/make/build_aarch64_lib.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # Copyright 2017 The TensorFlow Authors. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
@@ -20,5 +20,4 @@ set -e
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 TENSORFLOW_DIR="${SCRIPT_DIR}/../../../.."
 
-make -j 4 TARGET=aarch64 -C "${TENSORFLOW_DIR}" -f tensorflow/lite/tools/make/Makefile
-
+make TARGET=aarch64 -C "${TENSORFLOW_DIR}" -f tensorflow/lite/tools/make/Makefile $@
diff --git a/tensorflow/lite/tools/make/targets/aarch64_makefile.inc b/tensorflow/lite/tools/make/targets/aarch64_makefile.inc
index 0c38463bb6..e69de29bb2 100644
--- a/tensorflow/lite/tools/make/targets/aarch64_makefile.inc
+++ b/tensorflow/lite/tools/make/targets/aarch64_makefile.inc
@@ -1,34 +0,0 @@
-# Settings for generic aarch64 boards such as Odroid C2 or Pine64.
-ifeq ($(TARGET),aarch64)
-  # The aarch64 architecture covers all 64-bit ARM chips. This arch mandates
-  # NEON, so FPU flags are not needed below.
-  TARGET_ARCH := armv8-a
-  TARGET_TOOLCHAIN_PREFIX := aarch64-linux-gnu-
-
-  CXXFLAGS += \
-    -march=armv8-a \
-    -funsafe-math-optimizations \
-    -ftree-vectorize \
-    -fPIC
-
-  CFLAGS += \
-    -march=armv8-a \
-    -funsafe-math-optimizations \
-    -ftree-vectorize \
-    -fPIC
-
-  LDFLAGS := \
-    -Wl,--no-export-dynamic \
-    -Wl,--exclude-libs,ALL \
-    -Wl,--gc-sections \
-    -Wl,--as-needed \
-    -lrt
-
-       
-  LIBS := \
-    -lstdc++ \
-    -lpthread \
-    -lm \
-    -ldl
-
-endif
diff --git a/tensorflow/lite/tools/make/targets/generic_aarch64_makefile.inc b/tensorflow/lite/tools/make/targets/generic_aarch64_makefile.inc
index f4e4f1f9c4..e69de29bb2 100644
--- a/tensorflow/lite/tools/make/targets/generic_aarch64_makefile.inc
+++ b/tensorflow/lite/tools/make/targets/generic_aarch64_makefile.inc
@@ -1,33 +0,0 @@
-# Settings for generic aarch64 boards such as Odroid C2 or Pine64.
-ifeq ($(TARGET),generic-aarch64)
-  # The aarch64 architecture covers all 64-bit ARM chips. This arch mandates
-  # NEON, so FPU flags are not needed below.
-  TARGET_ARCH := armv8-a
-  TARGET_TOOLCHAIN_PREFIX := aarch64-linux-gnu-
-
-  CXXFLAGS += \
-    -march=armv8-a \
-    -funsafe-math-optimizations \
-    -ftree-vectorize \
-    -fPIC
-
-  CCFLAGS += \
-    -march=armv8-a \
-    -funsafe-math-optimizations \
-    -ftree-vectorize \
-    -fPIC
-
-  LDFLAGS := \
-    -Wl,--no-export-dynamic \
-    -Wl,--exclude-libs,ALL \
-    -Wl,--gc-sections \
-    -Wl,--as-needed
-
-       
-  LIBS := \
-    -lstdc++ \
-    -lpthread \
-    -lm \
-    -ldl
-
-endif
